.ci/azure-pipelines.yml

```diff
@@ -0,0 +1,183 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+variables:
+  - name: TestProjects
+    value: 'Jellyfin.Server.Tests/Jellyfin.Server.Tests.csproj'
+  - name: RestoreBuildProjects
+    value: 'Jellyfin.Server/Jellyfin.Server.csproj'
+
+pr:
+  autoCancel: true
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+
+jobs:
+  - job: main_build
+    displayName: Main Build
+    pool:
+      vmImage: ubuntu-16.04
```

Comment:
Why not 18.04?

Reply:
It does not exist.

Reply:
Here are the agents pools: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/pools-queues?view=azure-devops

Reply:
Ah damn.